cms/datasets/tests/test_utils.py

```diff
@@ -44,3 +52,149 @@ def test_format_datasets_as_document_list(self):
                 "description": f"<p>{manual_dataset['description']}</p>",
             },
         )
+
+    def test_extract_edition_from_dataset_url(self):
+        url = "/datasets/wellbeing-quarterly/editions/september/versions/9"
+        extracted_edition = extract_edition_from_dataset_url(url)
+        self.assertEqual(extracted_edition, "september")
+
+    def test_extract_edition_from_dataset_url_invalid(self):
```

Comment:
We don't check for the `ValueError` raised

Reply:
`ValueError` is no longer raised so it was removed from the test